sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -2359,6 +2360,33 @@ def test_pretty_Lambda():
     assert upretty(expr) == ucode_str
 
 
+def test_pretty_TransferFunction():
+    tf1 = TransferFunction(s - 1, s + 1, s)
+    assert upretty(tf1) == u"s - 1\n─────\ns + 1"
+    tf2 = TransferFunction(2*s + 1, 3 - p, s)
+    assert upretty(tf2) == u"2⋅s + 1\n───────\n 3 - p "
+    tf3 = TransferFunction(p, p + 1, p)
+    assert upretty(tf3) == u"  p  \n─────\np + 1"
+
+
+def test_pretty_Series():
+    tf1 = TransferFunction(x + y, x - 2*y, y)
+    tf2 = TransferFunction(x - y, x + y, y)
+    assert pretty(Series(tf1, tf2)) == \
+        '         ⎛   2    ⎞\n (x - y)⋅⎝x⋅y  - z⎠\n───────────────────\n⎛   3    3⎞        \n⎝- t  + y ⎠⋅(x + y)'
+    assert pretty(Series(-tf2, -tf1)) == \
+        '         ⎛     2    ⎞\n(-x + y)⋅⎝- x⋅y  + z⎠\n─────────────────────\n ⎛   3    3⎞         \n ⎝- t  + y ⎠⋅(x + y) '
+
+
+def test_pretty_Parallel():
+    tf1 = TransferFunction(x + y, x - 2*y, y)
+    tf2 = TransferFunction(x - y, x + y, y)
+    assert pretty(Parallel(tf1, tf2)) == \
+        '⎛   3    3⎞                   ⎛   2    ⎞\n⎝- t  + y ⎠⋅(x - y) + (x + y)⋅⎝x⋅y  - z⎠\n────────────────────────────────────────\n          ⎛   3    3⎞                   \n          ⎝- t  + y ⎠⋅(x + y)           '
+    assert pretty(Parallel(-tf2, -tf1)) == \
+        '⎛   3    3⎞                    ⎛   2    ⎞\n⎝- t  + y ⎠⋅(-x + y) - (x + y)⋅⎝x⋅y  - z⎠\n─────────────────────────────────────────\n           ⎛   3    3⎞                   \n           ⎝- t  + y ⎠⋅(x + y)           '
```

Comment:
It would be helpful if these were readable. For example you can do something like:

```
expected = """\
my
multi-line
thing
"""